python/ray/services.py

```diff
@@ -559,12 +554,10 @@ def wait_for_redis_to_start(redis_ip_address,
             redis_client.client_list()
         except redis.ConnectionError:
             # Wait a little bit.
-            time.sleep(1)
-            logger.info("Failed to connect to the redis server, retrying.")
-            counter += 1
+            time.sleep(0.001)
```

Comment:
Any reason not to just do exponential backoff here?

Reply:
No reason, just changed to use exponential backoff.